### DISCUSSION TITLE
"Denied background execution", what is this? How do I fix it?

### DISCUSSION THREAD

**Discussion by ySergi** [bkaankose/Wino-Mail#351] [Q&A]
![Screenshot 2024-09-04 121347](https://github.com/user-attachments/assets/1447a74e-24ac-4056-ba61-be60309e8651)

**Comment by bkaankose** [comment]
Go to start menu of Windows, locate Wino Mail in All Apps. Right click on "All Settings" and change this value to Always

![image](https://github.com/user-attachments/assets/49283071-684c-4b44-a0f9-254a8c862c4b)

**Reply by ySergi** [reply]
I don't see that option: 
![Screenshot 2024-09-04 215512](https://github.com/user-attachments/assets/63dd37f4-3389-4bfc-8e4d-fe63f2cd19f8)

**Comment by bkaankose** [comment]
This is not relevant with v2 migration anymore. New version uses different background sync mechanism without requiring background tasks.

Closing the discussion.
